src/herder/test/HerderTests.cpp

```diff
@@ -1332,7 +1326,7 @@ testSCPDriver(uint32 protocolVersion, uint32_t maxTxSize, size_t expectedOps,
             REQUIRE(scp.validateValue(seq, nomV.first, true) ==
                     SCPDriver::kInvalidValue);
 
-            auto balV = makeTxPair(herder, txSet0, ct, !signedBallots);
+            auto balV = makeTxPair(herder, txSet0, ct, false);
```

Comment:
`nomV` and `balV` are not needed anymore:
* they are testing the same thing
* it's simpler to just add a new case below for `BASIC`